### PR TITLE
feat: Support `serverlessv2_scaling_configuration.seconds_until_auto_pause`

### PR DESCRIPTION
**Pull request by magreenbaum**
## Description
Support `serverlessv2_scaling_configuration.seconds_until_auto_pause`.

## Motivation and Context
https://github.com/hashicorp/terraform-provider-aws/issues/40441

## Breaking Changes
No. 

## How Has This Been Tested?
- [x] I have updated at least one of the `examples/*` to demonstrate and validate my change(s)
- [x] I have tested and validated these changes using one or more of the provided `examples/*` projects

- [x] I have executed `pre-commit run -a` on my pull request

### CODE REVIEW

**Review by bryantbiggs — APPROVED** [review]
thank you!

**Comment by antonbabenko** [comment]
This PR is included in version 9.11.0 :tada:

File: examples/serverless/main.tf [hunk 1]
```diff
@@ -182,8 +182,9 @@ module "aurora_postgresql_v2" {
   enable_http_endpoint = true
 
   serverlessv2_scaling_configuration = {
-    min_capacity = 2
-    max_capacity = 10
+    min_capacity             = 0
+    max_capacity             = 10
+    seconds_until_auto_pause = 3600
```
Comment (schammah) [inline examples/serverless/main.tf:187]: any reason it was set for 1 hour?
console defaults to 5 mins

Comment (antonbabenko) [reply examples/serverless/main.tf:187]: This is just an example. Real values should be decided by users individually.

Comment (schammah) [reply examples/serverless/main.tf:187]: yes using this already, thanks, was just reffering what was the default in the console when adding this setting ;)

**Pull request merged**